pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -115,6 +115,8 @@ type Cluster struct {
 	ownerInfo          *k8sutil.OwnerInfo
 	isUpgrade          bool
 	arbiterMon         string
+	// list of mons to be failed over
+	monFailoverList sets.Set[string]
```

Comment:
Keeping the list name is generic and not specific to fail over related to mon path change

Reply:
How about this?
```suggestion
	monsToFailover sets.Set[string]
```